doc/script_commands.txt

```diff
@@ -1032,6 +1032,11 @@ will only execute once and will not execute if the map server reconnects
 to the char server later. Note that all those events will be executed upon
 scripts reloading.
 
+OnNPCUnload:
+
+OnNPCUnload will be execute when a NPC is unloaded.
+For example if @reloadscript or @reloadnpc is used.
```

Comment:
Maybe include all commands?
```suggestion
OnNPCUnload is called when @reloadscript, @reloadnpc, @unloadnpc or @unloadnpcfile is used
```

Reply:
Yeah, this probably would prevent some questiosn. I'll change that.